src/query/execute_statement.c

```diff
@@ -17724,3 +17758,305 @@ do_insert_checks (PARSER_CONTEXT * parser, PT_NODE * statement, PT_NODE ** class
 exit:
   return error;
 }
+
+MOP
+do_get_server_obj_id (DB_IDENTIFIER * server_obj_id, DB_OBJECT * server_class_mop, const char *server_name)
+{
+  return do_get_obj_id (server_obj_id, server_class_mop, server_name, "link_name");
+}
+
+int
+do_drop_server (PARSER_CONTEXT * parser, PT_NODE * statement)
+{
+  DB_OBJECT *server_class = NULL, *server_object = NULL;
+  DB_IDENTIFIER server_obj_id;
+  DB_VALUE class_name_val;
```

Comment:
I think `class_name_val` can be removed.

Reply:
I agree. I will remove it.